### ISSUE TITLE
Feature Request: Trigger Popup only when modifier key is pressed

### ISSUE THREAD

**Issue by pdelre** [TfTHacker/obsidian42-strange-new-worlds#79]
I love the SNW Indicators on Links while writing but I often find that my pointer lands on an indicator while my hands are on the keyboard (likely links at the end of lines/bullets). This leads to my writing view being blocked by the popup until I move my mouse, breaking a flow of writing.

Given the Obsidian Link behavior uses a Ctrl/Cmd+Hover to trigger the Link Preview popup, could a Ctrl/Cmd+Hover also be added to the SNW Indicator UX, either by default (UX breaking change) or with a plugin option?

**Comment by pdelre** [comment]
@TfTHacker Yes, it would cover my needs. Thank you

**Comment by Friedolito** [comment]
I come across this issue quite often too. 
I wan't to throw in my initial idea: Just delay the popup appearance a bit. So you wont trigger it accidentally so often. 
But the other options sound great as well! :) 

Thank you for developing @TfTHacker !

**Comment by KillyMXI** [comment]
I'm not sure whether I should open another issue.
I have the same problem - accidental pop-ups often stand in the way.
I was going to request a different solution though!

For me it would be enough to have adjustable delay slider.
If I stop the cursor over a number for long enough - then show me the pop-up.

No extra key to use (I may simply forget the feature is there in that case...), but it will still prevent accidental triggers.

**Comment by TfTHacker** [comment]
This is being worked on and will be in the next update.
